ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/PartyManagementClient.scala

```diff
@@ -46,6 +50,14 @@ final class PartyManagementClient(service: PartyManagementServiceStub)(
       .listKnownParties(PartyManagementClient.listKnownPartiesRequest)
       .map(_.partyDetails.map(PartyManagementClient.details)(collection.breakOut))
 
+  def getParties(
+      parties: OneAnd[Set, Ref.Party],
```

Comment:
we are already using `scalaz.tag` in scala bindings, so I think `OneAnd` is appropriate here. There is no reason to call this method with an empty set. If you want all known parties, use `listKnownParties`.

Reply:
However there are a bunch of other cases in scala bindings where `OneAnd[Set, X]` could have been used or just `Set`... so if you want to keep scala bindings consistent, let me know and I will remove `OneAnd` from the interface.